packages/clients/src/helpers/is-response.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Validates an object is of type Response without using `instanceof`.
```

Comment:
Maybe add  issue this https://github.com/scaleway/scaleway-sdk-js/issues/509 in comment for better understanding.